packages/mdc-fab/README.md

```diff
@@ -77,6 +77,29 @@ const fabRipple = new MDCRipple(document.querySelector('.mdc-fab'));
 
 > See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
+## Variants
+
+### Standard FAB
```

Comment:
Standard FAB is already expressed above, right? Variants only needs to include things not already explained under Basic Usage.